lib/kb_hmmer/Utils/HmmerUtil.py

```diff
@@ -1434,9 +1569,10 @@ def run_HMMER_Model_Group_Search(self, params):
                             elif many_type_name == 'Genome' or \
                                  many_type_name == 'AnnotatedMetagenomeAssembly' or \
                                  many_type_name == 'GenomeSet' or \
+                                 many_type_name == 'Tree' or \
                                  many_type_name == 'FeatureSet':
```

Comment:
isn't it just `if many_type_name != 'SequenceSet'`?

Reply:
at the moment, yes.  I wanted to make it explicit which types are being supported.